www/src/pages/components/fab/ios/index.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: Fab
+description: Buttons providing easy access to important actions.
+---
+
+import { graphql } from 'gatsby';
+import { ComponentHeader } from 'components/thumbprint-components';
+
+<ComponentHeader data={props.data} />
+
+## Summary
+
+There are two floating action button components: `TextFloatingActionButton` and `IconFloatingActionButton`.
+
+`TextFloatingActionButton` is used to display text and includes an option to add a `UIView` to the left of it.
+
+`IconFloatingActionButton` is an icon or image only floating action button.
+
+### Theme
+
+There are two standard fab themes on iOS. In addition, custom themes can be defined by calling the `TextFloatingActionButton.Theme` and `IconFloatingActionButton.Theme` constructors directly.
+
+#### TextFloatingActionButton Primary
+
+![Primary text fab](/img/ios/text_fab_primary.png)
+
+#### TextFloatingActionButton Secondary
+
+![Secondary text fab](/img/ios/text_fab_secondary.png)
+
+#### IconFloatingActionButton Primary
+
+![Primary icon fab](/img/ios/icon_fab_primary.png)
+
+#### IconFloatingActionButton Secondary
```

Comment:
I believe icon fab is meant to support only primary style. https://www.figma.com/file/YrpLNWoOwvEADbyFBWzuKv0V/Thumbprint-Global?node-id=1258%3A90

But I see web supports both as well. Maybe we should add support for it on android too? 🤷🏼‍♀️  @mallikapotter

Reply:
iOS supports both too- I think that figma might just not be complete. Does Android currently support the secondary style or does it need to be updated before it'd work?

Reply:
Android only supports primary style for IconFab based on the requirements here - https://docs.google.com/document/d/1yms-6TGSliYF0I3nyX5oxH-GOWORRgXyxJseVh0LJuM/edit
I'll let Mallika weigh in here.